src/MMLib.SwaggerForOcelot/Configuration/OcelotSwaggerGenOptions.cs

```diff
@@ -27,9 +27,16 @@ public class OcelotSwaggerGenOptions
         public bool GenerateDocsForGatewayItSelf { get; set; } = false;
 
         /// <summary>
-        /// Get or sets values indicating file paths of XML comment of gateway it self.
+        /// Generates docs for gateway it self with options.
         /// </summary>
-        public string[] FilePathsForXmlCommentsOfGatewayItSelf { get; set; }
```

Comment:
This is a breaking change. But Ok, because I don't assume anyone is already using it.

Reply:
Probably just me :)